zerver/models.py

```diff
@@ -4281,6 +4281,25 @@ class ScheduledMessageNotificationEmail(models.Model):
     scheduled_timestamp = models.DateTimeField(db_index=True)
 
 
+class StreamScheduledMessageAPI(TypedDict):
+    scheduled_message_id: int
+    to: int
+    type: str
+    content: str
+    rendered_content: str
+    topic: str
+    scheduled_delivery_timestamp: int
+
+
+class DirectScheduledMessageAPI(TypedDict):
```

Comment:
These could probably be named better?

Reply:
I think I agree but I'm not sure what to call it; let's discuss in a #backend thread as a follow-up.

Reply:
CZO topic: https://chat.zulip.org/#narrow/stream/3-backend/topic/scheduled.20message.20API.20dicts/near/1560746